README.md

```diff
@@ -10,21 +10,29 @@ You can reset the Vega Editor by going to https://vega.github.io/editor/#/reset
 
 ## Usage Instructions
 
-To run the editor locally, you must first install the dependencies and then launch a local web server. We assume you have [yarn](https://yarnpkg.com/) installed.
+### In Short
+```bash
+sudo docker run -it --rm -p 1234:1234 node:21 bash -xc 'mkdir -p vega/editor && git clone --depth=1 https://github.com/vega/editor.git vega/editor && cd vega/editor && yarn && yarn start'
+```
 
-1. Install the dependencies:
+### In Detail
+We assume you have the following tools installed:
+- [yarn](https://yarnpkg.com/)
+- The older `node` version 21, because of an incompatibility in one of our dependencies, see [node-canvas issue](https://github.com/Automattic/node-canvas/issues/2377)
```

Comment:
This feels out of place in the readme since we can hopefully fix this going forward and will forget to update it here. What if we set the node version in the package.json instead?

Reply:
Yeah, I thought the same too, but wasn't sure where to put it instead.

> What if we set the node version in the package.json instead?

If that is possible and an acceptable practice, that would be great. I have no experience with tools like node and yarn, and cannot judge that.  
I assumed there was a reason node wasn't already managed through package.json or yarn.lock, e.g. that the tools installing/updating the packages cannot modify the node installation while they run inside it.

I removed the hint without changing anything else. I'd be happy if you could handle that. But I'm also ok with merging as is. By reading the docker command, the information which node version to use is still there.

> hopefully fix this going forward

I was hoping so too. It seems there will be no release for node-canvas 2.x anymore; they already have a release preview for 3.0. So to make use of the upstream fix, vega-editor  probably has to upgrade to node-canvas 3.x once its officially released. I have no idea, if there are any breaking changes that could affect vega-editor.